Hashnode/mern-starter
# add scss support and make nodemon development server

SCSS support in development and production.

In dev, works by bundling with the javascript so that it works with the HMR.  In production, works by generating app.css using `sass` command then minifying to app.min.css and inserting into `client/index.js`. 

To test:
* confirm `npm start` works as intended
* confirm `npm run-script start:prod` works as intended
